### ISSUE TITLE
change of ip address requires new discovery.

### ISSUE THREAD

**Issue by robademar** [Aircoookie/Espalexa#112]
2 steps: 
1) I used espalexa to control my courtains.  Everything works fine , Alexa dicovers the device and it operates correctly. When  I power off esp32 and on again sometimes it will have a different ip address assigned and Alexa doesn't find it anymore. A new discovery cycle is required.
2) I then compiled and loaded the basic example and , if it will have the same ip address of the previous test, Alexa will operate it with the name of the previous test.
In other words if I use dhcp the all stuff becomes unreliable.
Is it correct? Then is there any check in the espalexa for the device name or everything is based on ip address?  thanks for the help  Gio

**Comment by robademar** [comment]
problem bypassed using fix  ip address.
